### PR TITLE
smallfile | Integrate Internal ElasticSearch

### PR DESCRIPTION
**Pull request by Avilir**
This PR is replacing PR #2249
Re-factoring the test to use internal elasticsearch server to used by the ripsaw

and copy all data from the internal ES to the Main ES.

	# Importing the ocs.elasticsearch module, and creating elasticsearch fixture
	  that will deploy elasticsearch as part of the test.
	# Saving the es info and replace with internal es info
	  in the end of the test, setting it back
	# Adding -- to rsh command to prevent deprecation
	# Adding copy function that copy all data from Internal to external ES
	# Changing the deviation percentage from 5 to 20
	# Adding warning message about the port forwarding
	# Show the results link at the end of the test

Signed-off-by: Avi Liani <alayani@redhat.com>

### CODE REVIEW

File: tests/e2e/performance/test_small_file_workload.py [hunk 1]
```diff
@@ -437,7 +510,7 @@ def test_smallfile_workload(self, ripsaw, file_size, files, threads,
 
         # wait for benchmark pods to get created - takes a while
         for bench_pod in TimeoutSampler(
-            120, 3, get_pod_name_by_pattern, 'smallfile-client', 'my-ripsaw'
+            240, 10, get_pod_name_by_pattern, 'smallfile-client', 'my-ripsaw'
```
Comment (shreekarSS) [inline tests/e2e/performance/test_small_file_workload.py:513]: pls use constants for 'my-ripsaw' https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L164

File: tests/e2e/performance/test_small_file_workload.py [hunk 2]
```diff
@@ -282,9 +329,9 @@ def aggregate_samples_results(self):
                         mean = np.mean(results[self.managed_keys[key]["name"]])
 
                         pct_dev = (st_deviation / mean) * 100
-                        if pct_dev > 5:
+                        if pct_dev > 20:
```
Comment (keesturam) [inline tests/e2e/performance/test_small_file_workload.py:332]: Any reason to change the deviation to 20% from 5%?

File: tests/e2e/performance/test_small_file_workload.py [hunk 3]
```diff
@@ -105,13 +119,45 @@ def add_key(self, key, value):
...
+                result = int_es.search(index=ind, body=query)
+            except ESExp.NotFoundError:
+                log.warning(f'{ind} Not found in the Internal ES.')
+                continue
+
+            log.debug(f'The results from internal ES for {ind} are :{result}')
+            log.info(f'Writing {ind} into main ES server')
+            for doc in result['hits']['hits']:
+                log.debug(f'Going to write : {doc}')
+                self.es.index(index=ind, doc_type='_doc', body=doc['_source'])
```
[the remark this answers is not in the document]
Comment (Avilir) [reply tests/e2e/performance/test_small_file_workload.py:149]: When you insert index into ES, if it does not exist, it create a new one.

File: tests/e2e/performance/test_small_file_workload.py (hunk 2 above)
Comment (Avilir) [reply tests/e2e/performance/test_small_file_workload.py:332]: For now, until we will have fix environment for BM & VMware so the results will be more persistent.

File: tests/e2e/performance/test_small_file_workload.py (hunk 1 above)
Comment (Avilir) [reply tests/e2e/performance/test_small_file_workload.py]: Change it.

File: tests/e2e/performance/test_small_file_workload.py [hunk 4]
```diff
@@ -507,6 +593,14 @@ def test_smallfile_workload(self, ripsaw, file_size, files, threads,
                     full_results.aggregate_host_results()
                     test_status = full_results.aggregate_samples_results()
                     full_results.write()
+
+                    # Creating full link to the kibana on the elastic search
```
[the remark this answers is not in the document]
Comment (Avilir) [reply tests/e2e/performance/test_small_file_workload.py]: Change it.

**Pull request merged**